--- glassfish3/glassfish/config/asenv.conf.orig	2013-04-23 10:05:17.414914794 +0200
+++ glassfish3/glassfish/config/asenv.conf	2013-04-23 10:11:36.854904902 +0200
@@ -59,10 +59,10 @@
 # directory.
 #
 
-AS_IMQ_LIB="../../mq/lib"
-AS_IMQ_BIN="../../mq/bin"
+AS_IMQ_LIB="/usr/share/glassfish/mq/lib"
+AS_IMQ_BIN="/usr/share/glassfish/mq/bin"
 AS_CONFIG="../config"
 AS_INSTALL=".."
 AS_DEF_DOMAINS_PATH="../domains"
 AS_DEF_NODES_PATH="../nodes"
-AS_DERBY_INSTALL="../../javadb"
+AS_DERBY_INSTALL="/usr/share/glassfish/javadb"
